### DISCUSSION TITLE
model feedback loops

### DISCUSSION THREAD

**Discussion by rkb965** [posit-conf-2023/vetiver#36] [Q&A]
This discussion is so interesting to me and so hard to wrap my head around the details of how to deal with design/ evaluation choices when there are feedback loops in a deployed prediction model — does anyone have recommendations for more reading on this topic?

**Comment by juliasilge** [comment]
I think this is SUCH an interesting topic and a really tough one for ML in real life. I'll share three resources I have found useful:

- ["When bias begets bias: A source of negative feedback loops in AI systems"](https://www.microsoft.com/en-us/research/blog/when-bias-begets-bias-a-source-of-negative-feedback-loops-in-ai-systems/) from the Microsoft research blog (includes links to the author's papers for further reading)
- Chapter 13 of [*Building Machine Learning Pipelines*](https://www.oreilly.com/library/view/building-machine-learning/9781492053187/) (you do have to get access to the whole book somehow, and it's maybe not so useful overall for you)
- ["A Classification of Feedback Loops and Their Relation to Biases in Automated Decision-Making Systems"](https://arxiv.org/abs/2305.06055) from earlier this year

I'll also share this [little bonus resource](https://ai.stackexchange.com/questions/22481/how-to-solve-the-dangerous-feedback-loops-in-machine-learning) (a Stack Exchange Q&A) that I think is nicely stated.
